docs/en/enterprise-edition/policy-reference/sast-policies/go-policies/sast-policy-262.adoc

```diff
@@ -0,0 +1,85 @@
+== Use of binding to all network interfaces
```

Comment:
- Binding to all network interfaces used 
- Usage of binding to all network interfaces